common/src/main/java/com/wynntils/functions/WarFunctions.java

```diff
@@ -22,4 +26,126 @@ public Double getValue(FunctionArguments arguments) {
             return Models.GuildWarTower.getRemainingTimeUntilVolley() / 1000d;
         }
     }
+
+    // These are static, and only change per war
+
+    public static class TowerOwnerFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            Optional<WarTowerState> initialTowerStateOpt = Models.GuildWarTower.getInitialTowerState();
+
+            if (initialTowerStateOpt.isEmpty()) return "-";
+
+            return initialTowerStateOpt.get().ownerGuild();
+        }
+    }
+
+    public static class TowerTerritoryFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            Optional<WarTowerState> initialTowerStateOpt = Models.GuildWarTower.getInitialTowerState();
+
+            if (initialTowerStateOpt.isEmpty()) return "-";
+
+            return initialTowerStateOpt.get().territory();
+        }
+    }
+
+    // Initial tower state functions
+
+    public static class InitialTowerHealthFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            // Technically this should be a long, but functions don't support longs
+            Optional<WarTowerState> initialTowerStateOpt = Models.GuildWarTower.getInitialTowerState();
+
+            if (initialTowerStateOpt.isEmpty()) return -1;
+
+            return (int) initialTowerStateOpt.get().health();
+        }
+    }
+
+    public static class InitialTowerDefenseFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            Optional<WarTowerState> initialTowerStateOpt = Models.GuildWarTower.getInitialTowerState();
+
+            if (initialTowerStateOpt.isEmpty()) return -1;
+
+            return initialTowerStateOpt.get().defense();
+        }
+    }
+
+    public static class InitialTowerDamageFunction extends Function<CappedValue> {
+        @Override
+        public CappedValue getValue(FunctionArguments arguments) {
+            // Technically this is a RangedValue, but functions don't support RangedValue
```

Comment:
Should we fix that?

Reply:
This should be fine for the time being imo.

Reply:
Ok.